scim2-sdk-common/src/test/java/com/unboundid/scim2/common/GenericScimResourceObjectTest.java

```diff
@@ -185,6 +186,44 @@ public void testStringMethods() throws ScimException
         Path.fromString("bogusPath")).isEmpty());
   }
 
+
+  /**
+   * Test {@link GenericScimResource#setSchemaUrns}.
+   */
+  @Test
+  public void testGenericScimResourceSchemaUrns()
```

Comment:
This was also moved from the SetSchemaTest, and converted to AssertJ's assertions.